src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -1761,5 +1828,19 @@ private void OnScreenshotAcquired(IntPtr data)
             screenshotAcquiredCallback?.Invoke(image);
             image.Dispose();
         }
+
+        private void OnContextMenuCustomized(IntPtr data, IntPtr menu)
+        {
+            WebContextMenu contextMenu = new WebContextMenu(menu, false);
+            contextMenuCustomizedEventHandler?.Invoke(this, new WebViewContextMenuCustomizedEventArgs(contextMenu));
```

Comment:
How about changing this?
you can avoid creating unnecessary objects.
```csharp
contextMenuCustomizedEventHandler?.Invoke(this, new WebViewContextMenuCustomizedEventArgs(new WebContextMenu(menu, false)));
```

Reply:
Done.